### ISSUE TITLE
Can't get the streamUrl of Live Stream

### ISSUE THREAD

**Issue by yiupin** [0xced/XCDYouTubeKit#528]

**Comment by vuvdv3101** [comment]
XCDYoutubeVideo.m 
Change to :
Line 253
`if (httpLiveStream != nil) {
            _streamURL = [NSURL URLWithString:httpLiveStream];
            streamURLs[XCDYouTubeVideoQualityHTTPLiveStreaming] = [NSURL URLWithString:httpLiveStream];
        }`

This is my solution. hope can help you 👍

**Comment by devbest555** [comment]
I cannot get the streamURL of Live video yet.
any idea?

**Comment by nid252** [comment]
@devbest555 
Have you found any solution for Live Videos? Facing the same issue.

**Comment by devbest555** [comment]
yes, facing the same issue yet.
not found the solution yet.
help me, please.

**Comment by vuvdv3101** [comment]
Did you try my code ?

**Comment by vuvdv3101** [comment]
> XCDYoutubeVideo.m
> Change to :
> Line 253
> `if (httpLiveStream != nil) { _streamURL = [NSURL URLWithString:httpLiveStream]; streamURLs[XCDYouTubeVideoQualityHTTPLiveStreaming] = [NSURL URLWithString:httpLiveStream]; }`
> 
> This is my solution. hope can help you 👍

apply this

**Comment by devbest555** [comment]
yes, Tried it already but same issue(streamURL = nil for only "LIVE" videos)

**Comment by vuvdv3101** [comment]
can you show your code line 253  in XCDYoutubeVideo.m ?

**Comment by devbest555** [comment]
yes, will do so.
Also,  If you use skype or telegram, it will be better to chat real-time
my skype : live:.cid.e5675d33daad7e3f
telegram : @fullstackwe

**Comment by nid252** [comment]
@devbest555  Please update here if you find any solution

**Comment by joseandroidengineer** [comment]
Any solution to this yet?
